meetup/tests.py

```diff
@@ -113,95 +154,70 @@ def test_invalid_url_returns_404(self):
 
   def test_validator_returns_true_for_valid_url(self):
     lookup_agent = self.lookup_agent_builder("http://meetup.com/papers-we-love/")
-    is_valid = lookup_agent.is_real_meetup()
+    is_valid = lookup_agent.url_exists()
     self.assertTrue(is_valid)
 
   def test_validator_returns_false_for_invalid_url(self):
     lookup_agent = self.lookup_agent_builder("http://meetup.com/this-is-not-a-meetup/")
-    is_valid = lookup_agent.is_real_meetup()
+    is_valid = lookup_agent.url_exists()
     self.assertFalse(is_valid)
 
-  def test_events_lookup_returns_event(self):
-    lookup_agent = self.lookup_agent_builder("http://meetup.com/papers-we-love/")
-    response = lookup_agent.get_response('events')
-    self.assertEqual(response.status_code, 200)
-
   def lookup_agent_builder(self, link):
```

Comment:
It's a good idea to pull helper methods like this up to the top of your test class, before the test cases.